test/component/CustomsDeclarationCancellationSpec.scala

```diff
@@ -113,233 +111,17 @@ class CustomsDeclarationCancellationSpec extends ComponentTestSpec with AuditSer
-    }
-
-    scenario("A non-CSP successfully submits a cancellation request on behalf of somebody with Customs enrolment having X-Client-ID request header to v1 api") {
-      Given("A Software House wants to submit a valid cancellation request and API Gateway provides X-Client-ID header only")
-      startMdgWcoDecService()
-      startApiSubscriptionFieldsService(apiSubscriptionKeyForXClientIdV1)
-      val request: FakeRequest[AnyContentAsXml] =
-        ValidCancellationRequestWithXClientIdHeader.withHeaders(RequestHeaders.ACCEPT_HMRC_XML_V1_HEADER).fromNonCsp.postTo(endpoint)
-
-      And("declarant is enrolled with Customs having an EORI number")
-      authServiceUnauthorisesScopeForCSP(nonCspBearerToken)
-      authServiceAuthorizesNonCspWithEori()
-
-      When("a POST request with data is sent to the API")
-      val result: Future[Result] = route(app = app, request).value
-
-      Then("a response with a 202 (ACCEPTED) status is received")
-      status(result) shouldBe ACCEPTED
-
-      And("the response body is empty")
-      contentAsString(result) shouldBe 'empty
-
-      And("the request was authorised with AuthService")
-      verifyAuthServiceCalledForNonCsp()
-
-      And("the api-subscription-fields service was called with value of X-Client-ID header and api version 1.0")
-      verifyGetSubscriptionFieldsCalled(apiSubscriptionKeyForXClientIdV1)
-      verifyGetSubscriptionFieldsNotCalled(apiSubsKey = apiSubscriptionKeyForXClientIdV2)
-    }
-
-    scenario("A non-CSP successfully submits a cancellation request on behalf of somebody with Customs enrolment having X-Client-ID request header to v2 api") {
-      Given("A Software House wants to submit a valid cancellation request and API Gateway provides X-Client-ID header only")
-      startMdgWcoDecService()
-      startApiSubscriptionFieldsService(apiSubscriptionKeyForXClientIdV2)
-      val request: FakeRequest[AnyContentAsXml] = ValidCancellationRequestWithXClientIdHeader.fromNonCsp.postTo(endpoint)
-
-      And("declarant is enrolled with Customs having an EORI number")
-      authServiceUnauthorisesScopeForCSP(nonCspBearerToken)
-      authServiceAuthorizesNonCspWithEori()
-
-      When("a POST request with data is sent to the API")
-      val result: Future[Result] = route(app = app, request).value
-
-      Then("a response with a 202 (ACCEPTED) status is received")
-      status(result) shouldBe ACCEPTED
-
-      And("the response body is empty")
-      contentAsString(result) shouldBe 'empty
-
-      And("the request was authorised with AuthService")
-      verifyAuthServiceCalledForNonCsp()
-
-      And("the api-subscription-fields service was called with value of X-Client-ID header and api version 2.0")
-      verifyGetSubscriptionFieldsNotCalled(apiSubscriptionKeyForXClientIdV1)
-      verifyGetSubscriptionFieldsCalled(apiSubsKey = apiSubscriptionKeyForXClientIdV2)
-    }
   }
 
   //***  UNHAPPY PATH SPECS ***
```

Comment:
remove comments
There is one scenario in each feature, better to have one feature with three scenarios?